### PR TITLE
feat(admin): Add Czech language support

### PR DESCRIPTION
**Pull request by hexrw**
- add Czech messages
- add Czech to supported languages
- add Czech to the docs

### CODE REVIEW

**Comment by CaseoJKL** [comment]
My team is trying to migrate an e-commerce to medusa and the language needs to be Czech. I would appreciate it if you can review and merge this translation.

**Comment by hexrw** [comment]
I have a dev team from our partner company also wanting to utilize Medusa for one of their projects. However, same as us, they depend on Czech translations being available.

File: www/apps/docs/content/user-guide/tips/languages.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ Languages listed in this section are contributed by the Medusa community. So, th
 :::
 
 - Bosanski
+- Čeština
```
Comment (kasperkristensen) [inline www/apps/docs/content/user-guide/tips/languages.md]: rm: This needs to be added in a separate PR, otherwise it gets published to our docs as soon as the PR is merged

**Review by kasperkristensen — CHANGES_REQUESTED** [review]
LGTM, just need to remove the docs change, and then we can merge it 🙂

**Review by hexrw — COMMENTED** [review]
Resolved conflicts

**Comment by hexrw** [comment]
I can still see the Vercel checks, if this did not stop the docs from deploying, should I make a new PR perhaps? However, the docs changes were removed and latest changes from `develop` were merged

**Comment by kasperkristensen** [comment]
> I can still see the Vercel checks, if this did not stop the docs from deploying, should I make a new PR perhaps? However, the docs changes were removed and latest changes from `develop` were merged

No it's fine, they just auto cancel when the PR doesn't include docs changes, and won't prevent me from merging once the other checks have passed 🙂

File: www/apps/docs/content/user-guide/tips/languages.md (hunk 1 above)
Comment (hexrw) [reply www/apps/docs/content/user-guide/tips/languages.md]: Could you mark those changes as resolved? It seems [a check is waiting for these changes to be confirmed](https://github.com/medusajs/medusa/pull/5822/checks?check_run_id=19751722388)

Comment (hexrw) [reply www/apps/docs/content/user-guide/tips/languages.md]: @kasperkristensen Could you mark the changes as resolved? It seems [a check is waiting for them to be resolved](https://github.com/medusajs/medusa/pull/5822/checks?check_run_id=19751722388)

**Review by kasperkristensen — APPROVED** [review]
